### PR TITLE
Add Bower support.

### PR DESCRIPTION
**Pull request by qw3rtman**
I created a barebones bower.json file. Additionally, I added a .gitignore for Bower. Confirm that the details in bower.json are correct.

(I have not registered the package.)

### CODE REVIEW

**Comment by lipis** [comment]
### WTF?!

screen shot 2015-05-06 at 01 39 29

**Comment by qw3rtman** [comment]
Seems I added the `.gitignore` for Bower after I populated `bower_components`, hence the flood of additions. :sweat_smile: 

Fixed and pushed one that is significantly lighter.

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "zoom.js",
+  "version": "0.0.1",
```
Comment (lipis) [inline bower.json]: No need for version here

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "homepage": "https://github.com/fat/zoom.js",
+  "authors": [
+    "Jacob Thornton <@fat>"
+  ],
+  "description": "Medium's Image Zoom for jQuery",
+  "main": "js/zoom.js",
```
Comment (cvrebert) [inline bower.json]: Missing `css/zoom.css`

**Comment by qw3rtman** [comment]
@cvrebert Added `css/zoom.css`.

File: bower.json (hunk 2 above)
Comment (qw3rtman) [inline bower.json]: @cvrebert Added `css/zoom.css`.

File: bower.json (hunk 1 above)
Comment (qw3rtman) [inline bower.json]: Why is that?

Comment (cvrebert) [inline bower.json]: Because https://github.com/bower/bower.json-spec/commit/a325da3d79baab018c572d75dc1781b12322f6cd

Comment (qw3rtman) [inline bower.json]: @cvrebert Ah, was not aware of that.

Removed `version`.

**Comment by djch** [comment]
So this installs the whole of Bootstrap as a dependency just for transition.js?

Could this be done with git submodule or something like that?

**Comment by qw3rtman** [comment]
I'm not familiar with submodules; however, I could register a Bower package that contains `transition.js`. Thoughts?

**Comment by djch** [comment]
Let's ask @cvrebert, I know he's way cleverer than me.

**Comment by fat** [comment]
i lol'd

**Comment by fat** [comment]
@qw3rtman thank's for opening this pr. 

I feel like i'm going to reject it for now tho.

If i add 1 for bower, that sorta opens the flood gates to all these packagemangers who can't figure out their shit.

I think I'm going to bank on the gulp/grunt model.

Which is - the community can just maintain a bower-zoom if they want like they have with gulp-postcss, etc.

Just want to keep this repo pure.

That might make 0 sense. But just trying to keep things as simple as possible with this tiny project.

Thanks though!

**Pull request closed without merging**